crates/uv/tests/pip_install_scenarios.rs

```diff
@@ -931,8 +931,12 @@ fn extra_incompatible_with_extra() {
 
     ----- stderr -----
       × No solution found when resolving dependencies:
-      ╰─▶ Because only package-a[extra-c]==1.0.0 is available and package-a[extra-c]==1.0.0 depends on package-b==2.0.0, we can conclude that all versions of package-a[extra-c] depend on package-b==2.0.0.
-          And because package-a[extra-b]==1.0.0 depends on package-b==1.0.0 and only package-a[extra-b]==1.0.0 is available, we can conclude that all versions of package-a[extra-c] and all versions of package-a[extra-b] are incompatible.
+      ╰─▶ Because only package-a[extra-c]==1.0.0 is available and package-a[extra-c]==1.0.0 depends on package-a[extra-c]==1.0.0, we can conclude that all versions of package-a[extra-c] depend on package-a[extra-c]==1.0.0.
```

Comment:
Epic

> Because only package-a[extra-c]==1.0.0 is available and package-a[extra-c]==1.0.0 depends on package-a[extra-c]==1.0.0, we can conclude that all versions of package-a[extra-c] depend on package-a[extra-c]==1.0.0.

Reply:
Yeah this is a real regression. Will look into it.